test_user.py

```diff
@@ -8,23 +8,23 @@ def test_when_player_is_created_it_has_a_symbol():
 def test_valid_choice_is_returned():
     game_board = Board()
     game_board.new_game()
-    assert spot_validation(game_board.board_state, "1") == 1
+    assert UserActions.make_move(game_board.board_state, "1") == 1
     
 def test_invalid_symbol_returns_false():
     game_board = Board()
     game_board.new_game()
-    assert spot_validation(game_board.board_state, "x") == False 
+    assert UserActions.make_move(game_board.board_state, "x") == False 
     
 def test_invalid_spot_too_large_returns_false():
     game_board = Board()
     game_board.new_game()
-    assert spot_validation(game_board.board_state, "12") == False
+    assert UserActions.make_move(game_board.board_state, "12") == False
 
 def test_when_a_user_finishes_move_it_is_the_opponents_turn():
     game_board = Board()
     game_board.new_game()
-    update_state(game_board.board_state, User.current_player, 1)
+    BoardState.update_state(game_board.board_state, User.current_player, 1)
     player1 = User("X")
     player2 = User("O")
-    turn_swap(User.current_player, player2.symbol)
+    User.switch_current_user(User.current_player, player2.symbol)
```

Comment:
Why does this function exist. It seems superfluous. Why should the user have to switch the current user. That doesn't seem like the user's job.

Reply:
You're right. Thats not the users job, it should be part of a Game class. I thought about putting this on the Board class, but that also seems inappropriate.